tests/tracer/test_forksafe.py

```diff
@@ -285,3 +286,59 @@ def fn():
     pid, status = os.waitpid(child, 0)
     exit_code = os.WEXITSTATUS(status)
     assert exit_code == 42
+
+
+@pytest.mark.subprocess(
+    out="" if (3,) < sys.version_info < (3, 7) else ("CTCTCT" if sys.platform == "darwin" else "CCCTTT"),
```

Comment:
is this consistent? I'd guess this was flaky.

do we just care that stdout contains 3 "C" and 3 "T" regardless of the order?

Reply:
wonder if it is just how darwin handles the scheduling

Reply:
It looks like it, but it "shouldn't be" 😕